package.json

```diff
@@ -3,6 +3,7 @@
   "version": "1.0.0",
   "private": true,
   "dependencies": {
+    "chai": "^4.1.0",
```

Comment:
I saw it in a video and several articles about testing components and things like componentDidMount(). It seems like it could be really helpful, but perhaps won't be totally used. Maybe I need to ask what other people are using...